inference-engine/ie_bridges/python/sample/classification_sample_async/classification_sample_async.py

```diff
@@ -158,9 +171,8 @@ def main():
         if len(output_queue) == 0:
             break
 
-# ----------------------------------------------------------------------------------------------------------------------
-    log.info('This sample is an API example, '
-             'for any performance measurements please use the dedicated benchmark_app tool\n')
+    # ----------------------------------------------------------------------------------------------------------------------
+    log.info('This sample is an API example, ' 'for any performance measurements please use the dedicated benchmark_app tool\n')
```

Comment:
strange fix

Reply:
change in the indent

Reply:
fixed, it is auto-formatter strange fix for line size 160